packages/slice-machine/src/icons/MenuBookIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import type { FC, SVGProps } from "react";
+
+export const MenuBookIcon: FC<SVGProps<SVGSVGElement>> = (props) => (
```

Comment:
We still need to add an icon like that manually for the side bar. Benj will take a topic in the future to switch to 100% editor-ui icons.